examples/summarization/bart/evaluate_cnn.py

```diff
@@ -37,7 +37,7 @@ def generate_summaries(
             min_length=min_length + 1,  # +1 from original because we start at step=1
             no_repeat_ngram_size=3,
             early_stopping=True,
-            decoder_start_token_id=model.config.eos_token_id,
+            bos_token_id=model.config.eos_token_id,
```

Comment:
Same as below, this might confuse people a lot and also pretty sure that this is not in line with Bart's original code. Bart has a weird hack in that it starts with the `eos_token_id` and then forces the `bos_token_id` which is different from the EOS token id  (@sshleifer) .

Reply:
rolled back decoder_start_token_id changes